internal/zero/leaser/restart.go

```diff
@@ -49,15 +48,14 @@ func restartContexts(
 	contexts chan<- context.Context,
 	restartFn func(context.Context) error,
 ) {
-	bo := backoff.NewExponentialBackOff()
-	bo.MaxElapsedTime = 0 // never stop
+	b := backoff.NewExponentialBackOff()
+	b.MaxElapsedTime = 0 // never stop
 
-	ticker := time.NewTicker(bo.InitialInterval)
+	ticker := backoff.NewTicker(b)
```

Comment:
maybe let's just change it to time.Timer() and do manual reset on each iteration?